filament/backend/src/metal/MetalDriver.mm

```diff
@@ -246,6 +252,10 @@
     return alloc_handle<MetalTexture, HwTexture>();
 }
 
+Handle<HwTexture> MetalDriver::importTextureS() noexcept {
+    return alloc_handle<MetalTexture, HwTexture>();
```

Comment:
I think returning a null handle via `return {}` here (and for the equivalent in the Vulkan backend) might be better. This is what we do for `createStreamFromTextureIdS`. Otherwise we're allocating a handle to a `MetalTexture` but never constructing it.

Reply:
I don't think this would be correct though. Because a null handle means "the allocation didn't work". i.e. the caller could assert() or something on it. 

Generally it is a mistake to allocate backend resources in `fooS`(which is called on the main thread), this must be done in `fooR`. So hopefully, this is not what the vulkan/metal backends do.